templates/zerver/emails/confirm_registration.source.html

```diff
@@ -6,10 +6,10 @@
 
 {% block content %}
 <p>
-    {{ _('You recently signed up for Zulip. Awesome!') }}
+    {{ _('You have requested a new Zulip organization. Awesome!') }}
 </p>
 <p>
-    {{ _('Click the button below to complete registration.') }}
+    {{ _('Click the button below to create the organization and register your account.') }}
     <a class="button" href="{{ activate_url }}">{{ _('Complete registration') }}</a>
```

Comment:
For type conditionals, you could surround lines with if-blocks like the following when needed
```html
{% if realm_creation %}
    {{ _('You recently signed up for Zulip. Awesome!') }}
{% else %}
    {{ _('You have requested a new Zulip organization. Awesome!') }}
{% endif %}
```

Reply:
@PIG208 thanks, I understood this, but the subject for the email is coming from a txt file for which I need to create a new txt file, how do I name that txt file?

Reply:
I think the if syntax should also apply to the txt file, so you could basically do it in a similar way instead of creating a new file.

Reply:
I've made changes to my existing PR, please review it and do let me know if somethings needs to be changed, and thanks @PIG208  for your support.